npm/builder.js

```diff
@@ -174,10 +178,32 @@ Builder.prototype.runTest = function (callback) {
   callback();
 };
 
-Builder.prototype.completeRelease = function (callback) {
+Builder.prototype.pushRelease = function () {
+  var result = child_process.execSync('git remote -v').toString('utf8')
+  var lines = result.split(/\r?\n/);
+  let remoteName;
+  for (let line of lines) {
+    if (line.includes('(push)') && line.includes('asciidoctor/asciidoctor.js.git')) {
+      remoteName = line.split('\t')[0];
+      break;
+    }
+  }
+  if (typeof remoteName === 'undefined') {
+    log.warn('Unable to find the remote name of the original repository asciidoctor/asciidoctor.js');
+    return false;
+  } else {
+    this.execSync('git push ' + remoteName + ' master');
+    this.execSync('git push ' + remoteName + ' --tags');
```

Comment:
I speak about `--tags`.
If in my fork I have created some custom tags when you do `git push original --tags`, you will publish my custom tags.